<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMS_Dialog</class>
 <widget class="QDialog" name="MMS_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Btn_send">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Tedt_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Tedt_number">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>32</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
